docs/topics/development/tests.rst

```diff
@@ -2,25 +2,41 @@
 Running Tests
 =============
 
-Run your tests like this::
+Run tests from outside the docker container with ``make``::
 
-    py.test
+    make test
 
-There's also a few useful makefile targets like ``test``, ``tdd`` and
-``test_force_db``::
+Other, more niche, test commands::
 
-    make test
+    make test_failed # rerun the failed tests from the previous run
+    make test_force_db # run the entire test suite with a new database
+    make tdd # run the entire test suite, but stop on the first error
```

Comment:
I copied this from the `Makefile` and it didn't mention that. Do we need to here?